data-prepper-plugins/common/src/main/java/com/amazon/situp/plugins/sink/FileSink.java

```diff
@@ -15,15 +13,13 @@
 import java.nio.file.Paths;
 import java.util.Collection;
 
-import static com.google.common.base.Preconditions.checkNotNull;
 import static java.lang.String.format;
 
 @SitupPlugin(name = "file", type = PluginType.SINK)
 public class FileSink implements Sink<Record<String>> {
-    private static final Logger LOG = LoggerFactory.getLogger(FileSink.class);
```

Comment:
Are these intentional changes ? these seem to be a problem of merge ?

Reply:
Problem of merge.